versions.props

```diff
@@ -4,7 +4,7 @@ com.google.errorprone:error_prone_annotations = 2.3.1
 com.google.errorprone:error_prone_core = 2.3.1
 com.google.errorprone:error_prone_test_helpers = 2.3.1
 com.google.guava:guava = 23.6.1-jre
-com.netflix.nebula:nebula-dependency-recommender = 6.1.4
+com.netflix.nebula:nebula-dependency-recommender = 7.0.0
```

Comment:
Apparently this is a non-scary upgrade, it just bumped the minimum gradle version to 4.0+:

https://github.com/nebula-plugins/nebula-dependency-recommender-plugin/compare/v6.1.4...master